src/pages/workspace/accounting/qbo/import/QuickbooksTaxesPage.tsx

```diff
@@ -42,12 +45,14 @@ function QuickbooksTaxesPage({policy}: WithPolicyProps) {
                             <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
                                 <Switch
                                     accessibilityLabel={translate('workspace.accounting.taxes')}
-                                    isOn={!!syncTax}
+                                    isOn={isTaxSwitchOn}
                                     onToggle={() => Connections.updatePolicyConnectionConfig(policyID, CONST.POLICY.CONNECTIONS.NAME.QBO, CONST.QUICK_BOOKS_CONFIG.SYNC_TAX, !syncTax)}
+                                    disabled={isJournalExportEntity}
```

Comment:
this means that the tax toggle is not active when the journal entry is selected as the export account for the reimbursable expenses, right?

so we display an error but we keep the toggle button off?